### PR TITLE
Docs: update "OAT" guide to "Introduction" section

### PR DESCRIPTION
**Pull request by fakepixels**
**What changed? Why?**
Update "OAT" guide to "Introduction" section

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: site/docs/pages/build-your-first-onchain-app.mdx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+<img
+  alt="Onchain app template"
+  src="https://onchainkit.xyz/assets/Onchain-app-template-1.png"
+  width="auto"
+/>
+
+::::steps
+
+### Install
```
Comment (Zizzamia) [inline site/docs/pages/build-your-first-onchain-app.mdx:16]: I don't think we need this step, as the developer will:
1. click `use this template` or `fork`
2. run `bun i`, which will take care of installing OnchainKit

File: site/docs/pages/build-your-first-onchain-app.mdx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  src="https://onchainkit.xyz/assets/getting-started-create-env-file.png"
+  width="250"
+/>
+
+```tsx [.env]
+PUBLIC_ONCHAINKIT_API_KEY=YOUR_PUBLIC_API_KEY
+PUBLIC_WALLETCONNECT_PROJECT_ID=YOUR_WALLETCONNECT_PROJECT_ID
+```
+
+### Follow the rest of the set up in [Get Started](/getting-started) page.
```
Comment (Zizzamia) [inline site/docs/pages/build-your-first-onchain-app.mdx:71]: Some of the Getting Started step, might not necessary, as they are already part of the template. So we might want to only teach each how to get the Public Key and Wallet Connect project id.

File: site/docs/pages/build-your-first-onchain-app.mdx (hunk 1 above)
Comment (fakepixels) [reply site/docs/pages/build-your-first-onchain-app.mdx:16]: Cool!

File: site/docs/pages/build-your-first-onchain-app.mdx (hunk 2 above)
Comment (fakepixels) [reply site/docs/pages/build-your-first-onchain-app.mdx:71]: Sgtm!

**Comment by Zizzamia** [comment]
Thank you @fakepixels for starting this PR, I took inspiration from some of the content you wrote and mix it with the Build Onchain apps wording and published this https://onchainkit.xyz/guides/build-onchain-apps 

The wording is a mix of what we have in BOAT, so that Google will not freak out. And we can keep some of the SEO juice.

**Pull request closed without merging**